packages/plugin-conversation/src/conversation.js

```diff
@@ -421,11 +428,10 @@ const Conversation = SparkPlugin.extend({
    * @param {Event} event
    * @returns {Promise}
    */
-  processActivityEvent(event) {
-    return this.decrypter.decryptObject(event.activity)
-      .then(() => this.inboundNormalizer.normalize(event.activity))
-      .then(() => event);
-  },
+  // processActivityEvent(event) {
```

Comment:
actually, no, thats a really important function to put back.  i'll add a test to prove it works.